style2.scss

```diff
@@ -0,0 +1,315 @@
+html {
```

Comment:
Please include one line of space between CSS rules/blocks for better readability and legibility. I think Chiamaka has an awesome article that covers several topics including code format config with tools like Prettier and ESlint. I'll share a link on the group channel .